consensus/checks.go

```diff
@@ -197,6 +197,9 @@ func (consensus *Consensus) onPreparedSanityChecks(
 
 func (consensus *Consensus) viewChangeSanityCheck(msg *msg_pb.Message) bool {
 	consensus.getLogger().Debug().
+		Uint64("blockNum", msg.GetConsensus().BlockNum).
```

Comment:
We should remove this part from master. I only fixed it in t3.

Reply:
This PR is for t3, so we are good.

Reply:
fixed